test/scripts/base-host-check.sh

```diff
@@ -80,17 +80,20 @@ get_oscap_score() {
 }
 
 check_ca_cert() {
-    serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2-)
+    serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2- | tr '[:upper:]' '[:lower:]')
+    cn=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -subject | sed 's/.*CN=//' | sed 's/ /_/g')
 
-    echo "📗 Checking CA cert anchor file"
+    echo "📗 Checking CA cert anchor file serial '${serial}'"
     if ! [ -e "/etc/pki/ca-trust/source/anchors/${serial}.pem" ]; then
-        echo "Anchor CA file does not exist"
+        echo "Anchor CA file does not exist, directory contents:"
+        find /etc/pki/ca-trust/source/anchors
         exit 1
     fi
 
-    echo "📗 Checking extracted CA cert file"
-    if ! [ -e "/etc/pki/ca-trust/source/extracted/pem/directory-hash/Test_CA_for_osbuild.pem.pem" ]; then
-        echo "Extracted CA file does not exist"
+    echo "📗 Checking extracted CA cert file named '${cn}'"
+    if ! [ -e "/etc/pki/ca-trust/extracted/pem/directory-hash/${cn}.pem" ]; then
+        echo "Extracted CA file does not exist, directory contents:"
+        find /etc/pki/ca-trust/extracted/pem/directory-hash
         exit 1
     fi
 }
```

Comment:
@achilleas-k fyi this test was introduced last week but it was broken (a typo in `directory-hash` path) and due to the config-map it was not executed properly so it sneaked into `main`. This fixes the map and the test.

Reply:
Thank you!